src/index.ts

```diff
@@ -21,6 +24,17 @@ client.once(Events.ClientReady, async (bot) => {
 		console.error(`Error loading commands ${err}`);
 	}
 
+	// Initialize all event listeners
+	try {
+		for (const listenerName in listeners) {
+			const listener = listeners[listenerName as keyof typeof listeners]
+			listener(bot)
+		}
+	} catch (err) {
+		console.error(`Error loading listeners!\n${err}`);
+	}
+
```

Comment:
What about instead of wrapping the whole `for` loop inside a `try catch` block, just not wrap the addition?

```ts
		for (const listenerName in listeners) {
                        try {
			    listeners[listenerName as keyof typeof listeners](bot);
                        catch (e) {
                            console.error(`Error loading ${listenerName} due to ${e}`);
                        }
		}
```

And also standardize the usage of `;`, maybe.